designs/2022-docs-information-architecture-update/README.md

```diff
@@ -0,0 +1,252 @@
+- [MOVE & RENAME PAGE] Request a Change
+  - Current page: [Change Requests](https://eslint.org/docs/latest/developer-guide/contributing/changes)
+- [MOVE & RENAME PAGE] Work on Issues
+  - Current page: [Working on Issues](https://eslint.org/docs/latest/developer-guide/contributing/working-on-issues)
+- [NEW PAGE] Report Security Vulnerabilities
+  - Tales current content from the [Reporting a Security Vulnerability section on the Contributing page](https://eslint.org/docs/latest/developer-guide/contributing/#reporting-a-security-vulnerability). Worth breaking out into its own page for discoverability and SEO for an important subject.
+  - NOTE: Should be added, but not part of the initial IA project. Probably can be done as a one-off task or group in with one of the project phases.
+- [MOVE & RENAME PAGE] Submit a Pull Request
+  - Current page: [Pull Requests](https://eslint.org/docs/latest/developer-guide/contributing/pull-requests)
+
+## Drawbacks
+
+Potential drawbacks of this documentation update include:
+
+- Breaking user experience from the current documentation
+- Users potentially do not find the new information architecture more clear
+- Opportunity cost. Time spent on the information architecture updates could be spent on more impactful documentation tasks.
+
+<!--
+    Why should we *not* do this? Consider why adding this into ESLint
+    might not benefit the project or the community. Attempt to think
+    about any opposing viewpoints that reviewers might bring up.
+
+    Any change has potential downsides, including increased maintenance
+    burden, incompatibility with other tools, breaking existing user
+    experience, etc. Try to identify as many potential problems with
+    implementing this RFC as possible.
+-->
+
+## Backwards Compatibility Analysis
+
+HTTP redirects from existing content URL paths to the new content URL paths must be included as part of this project.
+
+<!--
+    How does this change affect existing ESLint users? Will any behavior
+    change for them? If so, how are you going to minimize the disruption
+    to existing users?
+-->
+
+## Alternatives
+
+- Do not change the IA. Only update documentation within the existing IA as part of the [documentation update project](https://github.com/eslint/eslint/issues/16365).
+
+<!--
+    What other designs did you consider? Why did you decide against those?
+
+    This section should also include prior art, such as whether similar
+    projects have already implemented a similar feature.
+-->
+
+## Open Questions
+
+1. Is there a way to automatically change all the internal links that'll be broken by these updates?
+1. Can you create a side navigation folder that is not also a page?
+1. How to make section landing pages navigable from the side table of contents? (For example, the proposed page "Use ESLint in Your Project landing page")
```

Comment:
sure. i think it's best illustrated by an example. in the current ESLint site, the page https://eslint.org/docs/latest/user-guide/ cannot be accessed from the left-hand side navigation.

Reply:
My previous comment actually makes more sense here. 😄 

> We have them currently because there are existing links out there that point to those pages but we don’t actually link them from the sidebar.

This was also a design decision because, for accessibility purposes, there isn’t a clear way to indicate that something is both a link and expandable. If we didn’t already have landing pages for user guide, etc., then we wouldn’t make them.

Reply:
i'll add a note about this to the RFC as well